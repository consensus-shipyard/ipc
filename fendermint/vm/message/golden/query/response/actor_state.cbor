a564636f6465d82a582300122086f2a6fa7d8fe1d53e08a2c67ef12ef6db77fedd6fb226f4274e42e967babb6d657374617465d82a5823001220ca8da62b1bc99224a0aeb7c8084d599e73d9a5a178ba22a00028d0a8b58f45e16873657175656e63651bfb6d7403eeb008af6762616c616e6365584100f51bc89d095b7349acf640d64d6ab3626af87a6fdfb058eac20a7d653ab35cb812f7e2f81d977fe9eb52fdadd27f138b269215958b8093b34caa7d54559423ef7164656c6567617465645f616464726573735831037aac7201ba99bde23adf83c6c4755500ffc48707223e8c714a2447ac7ea8d9112b7002422f91ac3b56775d995ef91377