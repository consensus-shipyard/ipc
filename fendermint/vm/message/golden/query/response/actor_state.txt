ActorState { code: Cid(QmXRRwhd1nN8zzfwsDhwQU5VeYf96rzb5NrJYr2bgz59HW), state: Cid(QmbyLLet5eDJxHPgXbRr146mgwtGWFkii3HvrqXDJpRkGY), sequence: 18117264436219611311, balance: TokenAmount(12837375353358104575764595313077765395794546135086142370656331495972793855553867421435159587158883947028703783392634027814199088685207472.895591517170967535), delegated_address: Some(Address { payload: BLS([122, 172, 114, 1, 186, 153, 189, 226, 58, 223, 131, 198, 196, 117, 85, 0, 255, 196, 135, 7, 34, 62, 140, 113, 74, 36, 71, 172, 126, 168, 217, 17, 43, 112, 2, 66, 47, 145, 172, 59, 86, 119, 93, 153, 94, 249, 19, 119]) }) }